plotmodel.py

```diff
@@ -308,6 +310,14 @@ def getDomains(self, file, type_):
             domain = DomainView(id, name, color, masked, highlighted)
             domains[id] = domain
 
+        # always add void to a material domain at the end
+        if 'material' in file:
+            void_id = str(_VOID_REGION)
+            domains[void_id] = DomainView(void_id, "VOID",
+                                          (255, 255, 255),
```

Comment:
Do we want to be able to have the color be configurable?

Reply:
Hrmmm I think it's as configurable as any other color in the plotter. It just doesn't get a random color assigned to start with - always defaults to white. Am I missing something?

Reply:
Nope, you're right. Thanks for pointing that out!